core/frontend/src/render/MockRender.ts

```diff
@@ -34,7 +34,8 @@ import { Scene } from "./Scene";
  *  (1) If overriding anything in the implementation supplied herein, pass a SystemFactory function to MockRender.App.systemFactory.
  *  (2) Call MockRender.App.startup() instead of IModelApp.startup() before tests begin.
  *  (3) Likewise call MockRender.App.shutdown() when finished. This resets the SystemFactory to its default.
- * @internal
+ * @note The APIs within this namespace are intended *strictly* for use with unit tests.
+ * @public
```

Comment:
do we really want to mark testing related apis as public?

Reply:
Everybody's using it all over the place, which I took as evidence of its usefulness.
The APIs it mocks are all `public`.

Reply:
That is correct, everybody's using it for tests, and I agree, moving them to a separate testing package would be ideal. Do you have someone in mind who can put together the new package for 4.0?

I disagree about it being ok to use internal APIs in tests outside of the itwinjs-core repo. It doesn't matter how an API is being used, if it's not public then it's subject to breakage at any time.

Reply:
Reverted to `internal`. If you want a new package, file an issue.

Reply:
Sorry was stuck in a meeting, i will discuss with others and see if we can find someone to create a testing pkg for 4.0